src/foam/nanos/controller/ApplicationController.js

```diff
@@ -137,6 +138,13 @@ foam.CLASS({
   `,
 
   properties: [
+    {
+      class: 'Int',
```

Comment:
I think this should be an Enum of the DisplayWidth that you just created. Your factory can use the window.innerWidth to compute the DisplayWidth that should be used.

Reply:
Done.